style-rtl.css

```diff
@@ -2463,6 +2476,12 @@ body:not(.singular) main article:first-of-type {
 	text-transform: uppercase;
 }
 
+.entry-categories a:focus,
+.entry-categories a:hover {
+	text-decoration: underline;
+	border-bottom: none;
```

Comment:
You can remove the `text-decoration: underline` here, and set it to `border-bottom-color: transparent` instead, so the elements beneath the link doesn't jump when the border width gets removed.

Reply:
@andersnoren With https://github.com/WordPress/twentytwenty/pull/548/commits/5d74df15fea5118d386a41e65eb7ab756b58f4de I've implemented your feedback. Feel free to test this PR again. 😀